src/cpp/server/audit_logging.h

```diff
@@ -0,0 +1,89 @@
+//
+//
+// Copyright 2023 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//
+
+#ifndef GRPC_SRC_CPP_SERVER_AUDIT_LOGGING_H
+#define GRPC_SRC_CPP_SERVER_AUDIT_LOGGING_H
+
+#include <memory>
+#include <string>
+#include <utility>
+
+#include "absl/status/statusor.h"
+#include "absl/strings/string_view.h"
+
+#include <grpc/grpc_audit_logging.h>
+#include <grpcpp/security/audit_logging.h>
+
+#include "src/core/lib/json/json.h"
+
+namespace grpc {
+namespace experimental {
+
+// Unregisters an audit logger factory. This should only be used in tests.
+void UnregisterAuditLoggerFactory(absl::string_view name);
```

Comment:
Same things about it being TestOnly, would move that to the beginning of the comment

Reply:
This function is gone now. But for the member function of the registry, I added `TestOnly` prefix.